src/pylorax/api/compose.py

```diff
@@ -135,9 +139,13 @@ def write_ks_root(f, user):
         if any(user["password"].startswith(prefix) for prefix in ["$2b$", "$6$", "$5$"]):
             log.debug("Detected pre-crypted password")
             f.write('rootpw --iscrypted "%s"\n' % user["password"])
+            wrote_rootpw = False
```

Comment:
shouldn't this (and the other one) be `wrote_rootpw = True` ?

Reply:
Doh.